core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -58,8 +58,7 @@
   // AppEngine runtimes have constraints on threading and socket handling
   // that need to be accommodated.
   public static final boolean IS_RESTRICTED_APPENGINE =
-      System.getProperty("com.google.appengine.runtime.environment") != null
-          && "1.7".equals(System.getProperty("java.specification.version"));
+      System.getProperty("com.google.appengine.runtime.environment") != null;
```

Comment:
The cipher is weaker if the iv is weaker.  

Actually, from reading what SSLContext.getInstance does, neither of "Default" or "TLS" seems right.  TLS could pick 1.1 or lower, which is forbidden by http2.  Default could pick SSLv3.


I don't actually see a protocol that says 1.2 or higher.

https://docs.oracle.com/javase/6/docs/technotes/guides/security/StandardNames.html#SSLContext